packages/stories-block/src/block/block-types/single-story/edit.js

```diff
@@ -107,9 +126,7 @@ function StoryEmbedEdit({
 
         setCannotEmbed(!(typeof data?.title === 'string'));
         setStoryData(data);
-        setAttributes({
-          url: localURL,
-        });
+        setAttributes({ url });
```

Comment:
We didn't need to use `localURL` here, as it will be the same value of url. It better for the callback.